nlp/Dockerfile

```diff
@@ -4,13 +4,48 @@ FROM birdhouse/pavics-jupyter-base:0.2.1
 USER root
 
 COPY environment.yml /environment.yml
+COPY /nl2query ./nl2query 
+COPY notebook_config.yml /notebook_config.yml
+
 
 # update env "birdy"
 # use umask 0000 so that package files for the updated environment are usable by the user for the jupyter-conda-extension
 RUN umask 0000 && conda env update -f /environment.yml
 
-COPY notebook_config.yml /notebook_config.yml
 
-# specify user because of problem running start-notebook.sh when being root
-USER jenkins
 
+# Installing java opjdk
+RUN conda install -c anaconda openjdk
+ENV LANG=C.UTF-8
+
+
+# Download spacy model
+RUN python -m spacy download en_core_web_trf
+
+# Heideltime Tree-tagger Installation 
+RUN cd nl2query/heideltime && \
```

Comment:
please specify `/nl2query` here, since `WORKDIR` instruction is not used. That will ensure we don't have path issues and always refer to the nl2query in the `/` path.